python/batch_local.py

```diff
@@ -0,0 +1,162 @@
+"""
+batch.py :: a script for launching sim jobs
```

Comment:
What's the difference between this and batch.py? Can we get rid of one?

Reply:
batch.py has parameters specific for NET3 and whichever cluster used slurm. This was to have a script which could run locally instead of on the servers. I was planning to add that additional feature into batch.py at some point. I'll do that.